gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #82 User Profile: Music Genres

### Description

I added a genres field to the CustomUser model and I updated the Edit Profile page so that a user can specify which genres they like. I also edited the View User Profile page so that you can see which genres the specified user likes.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
```

1. Get the code from this PR
1. Run `pip install django-materializecss-form`
1. Run the Django server, sign up, log in
1. Go to your user profile, click on "Edit Your Profile"
1. Write a nice Bio and click the checkboxes next to your favorite genres
> ![image](https://user-images.githubusercontent.com/13837978/68003207-bec2f680-fc42-11e9-802d-d0c9fc8d7ec4.png)
6. Click "Submit", see that you're taken back to your user profile and that your bio and genres are now shown, as desired
> ![image](https://user-images.githubusercontent.com/13837978/68003330-47da2d80-fc43-11e9-8669-13cf4816b4ff.png)
7. Sign in as a different user and go to your original user's profile and see that the bio and genres are visible to this other user, as desired



### Miscellaneous Comments

* I'm trying to use descriptive commit messages so please read them. They might pre-emptively answer some of your questions.

### Checklist
